config/config.md

```diff
@@ -162,6 +162,9 @@
 | `logging.log_format` | String | `text` | The log format. Can be `text`/`json`. |
 | `logging.tracing_sample_ratio` | -- | -- | The percentage of tracing will be sampled and exported.<br/>Valid range `[0, 1]`, 1 means all traces are sampled, 0 means all traces are not sampled, the default value is 1.<br/>ratio > 1 are treated as 1. Fractions < 0 are treated as 0 |
 | `logging.tracing_sample_ratio.default_ratio` | Float | `1.0` | -- |
+| `logging.slow_query` | -- | -- | The slow query log options. |
+| `logging.slow_query.enable` | Bool | `false` | Whether to enable slow query log. |
+| `logging.slow_query.threshold` | String | Unset | The threshold of slow query. |
```

Comment:
Except the threshold, maybe we need a sample rate too. Just like mongodb 

https://www.mongodb.com/zh-cn/docs/cloud-manager/performance-advisor/slow-query-threshold/

Reply:
Maybe it's not a necessary configuration? The current implementation is very cheap: log the slow query into file and collect by 3rd tools.

Reply:
The sample ratio determines whether to print all slow queries, defaulting to 1 (100%). If all queries are slow, printing them all could harm database performance.